src/App.tsx

```diff
@@ -21,42 +24,75 @@ const routes = [
             </RootLayout>
         ),
     },
+
     {
```

Comment:
Please comment out this route and leave a comment that it no longer works and that we should return to this later.

Reply:
Included comment for each route with specific to the page 
Disabled unused routes and added comments appropriately